projects/npm-tools/packages/npm-scripts/src/scripts/webpack.js

```diff
@@ -15,37 +16,35 @@ const WEBPACK_DEV_CONFIG_FILE = 'webpack.config.dev.js';
 /**
  * Main function for running webpack within the liferay-portal repo.
  */
-module.exports = function (...args) {
-	const watch = args.indexOf('--watch');
-	if (watch !== -1) {
+module.exports = function (...argv) {
+	const extraArgs = argv.filter(
+		(arg) => !['--federation', '--watch'].includes(arg)
+	);
+	const args = parseArgs(argv);
```

Comment:
I'd just skip `minimist` here, instead of calling it once in `src/index.js` and immediately after again here.

```js
const federation = argv.includes('--federation');
const watch = argv.includes('--watch');
```

Also, tiny nit: quadratic loop of `filter` + `includes`. I know it's not a performance problem because N is small, but just to avoid the pattern becoming normalized and then people replicating in other places where it actually does matter, I make a point of note writing nested loops like this. Instead, you could do something like:

```js
const SPECIAL_ARGS = new Set(['--federation', '--watch']);

module.exports = function (...args) {
   const extraArgs = args.filter((arg) => !SPECIAL_ARGS.has(arg));
}
```

(I don't know if the names are right, just an example).

One other minor thing, this thing that you're calling `argv` isn't actually `argv`: `argv` is the array of arguments passed to the executable when invoked — what you're getting here is a subset of those, so technically the name `argv` is giving a false impression of its contents.